CONTRIBUTING.md

```diff
@@ -2,12 +2,24 @@
 
 You are encouraged to contribute to the repository by **forking and submitting a pull request**.
 
-For significant changes, please open an issue first to discuss the proposed changes to avoid re-work.
-
 (If you are new to GitHub, you might start with a [basic tutorial](https://help.github.com/articles/set-up-git) and check out a more detailed guide to [pull requests](https://help.github.com/articles/using-pull-requests/).)
 
 Pull requests will be evaluated by the repository guardians on a schedule and if deemed beneficial will be committed to the main branch. Pull requests should have a descriptive name and include an summary of all changes made in the pull request description.
 
-If you would like to propose a significant change, please open an issue first to discuss the work with the community.
+If you would like to propose a significant change, please open an issue first to discuss the proposed changes with the community and to avoid re-work.
 
 Contributions are made pursuant to the Developer's Certificate of Origin, available at [https://developercertificate.org](https://developercertificate.org), and licensed under the Apache License, version 2.0 (Apache-2.0).
+
+## Contributing Lessons Learned
+
+### From our 0.2.0 release cycle, we learned:
+
+- It is difficult to manage a release with too many changes.
+  - We should **release weekly**, not monthly.
```

Comment:
I would word this as we should release often, not months apart or something (I don't think we necesarily need to release weekly)